ConsoleFramework/Controls/PasswordBox.cs

```diff
@@ -0,0 +1,291 @@
+﻿using System;
+using System.Collections.Generic;
+using ConsoleFramework.Core;
+using ConsoleFramework.Events;
+using ConsoleFramework.Native;
+using ConsoleFramework.Rendering;
+
+namespace ConsoleFramework.Controls
+{
+    public class PasswordBox : Control
```

Comment:
I think it will be good to try to inherit PasswordBox from TextBox and avoid some copypaste code

Reply:
It was old code I had written a long time ago I will fix it and open a new request